[09-Jul-2021 21:46:51 Europe/Berlin] PHP Warning:  preg_match(): Compilation failed: missing closing parenthesis at offset 26 in D:\XAMPP\htdocs\api\Core\Router.php on line 145
[09-Jul-2021 21:52:50 Europe/Berlin] PHP Warning:  preg_match(): Compilation failed: missing closing parenthesis at offset 26 in D:\XAMPP\htdocs\api\Core\Router.php on line 145
[09-Jul-2021 21:53:37 Europe/Berlin] PHP Warning:  preg_match(): Compilation failed: missing closing parenthesis at offset 26 in D:\XAMPP\htdocs\api\Core\Router.php on line 145
[09-Jul-2021 21:55:27 Europe/Berlin] PHP Fatal error:  Uncaught Error: Non-static method Resources\Table\TableModel::getAllTables() cannot be called statically in D:\XAMPP\htdocs\api\Resources\Table\TableController.php:10
Stack trace:
#0 [internal function]: Resources\Table\TableController->index('4')
#1 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#2 D:\XAMPP\htdocs\api\Core\Application.php(134): Core\Router->resolve()
#3 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#4 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Table\TableController.php on line 10
[09-Jul-2021 22:45:55 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to a member function where() on int in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php:31
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold('1', '2')
#1 [internal function]: Resources\Table\TableController->hold('1', '2')
#2 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#3 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#4 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#5 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php on line 31
[09-Jul-2021 22:50:21 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to a member function where() on int in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php:31
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#1 [internal function]: Resources\Table\TableController->hold('1', '2')
#2 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#3 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#4 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#5 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php on line 31
[09-Jul-2021 23:05:53 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:492
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(492): PDO->prepare('update `tables`...')
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('update `tables`...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#6 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#7 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#8 [internal function]: Resources\Table\TableController->hold('1', '2')
#9 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#10 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#11 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#12 {main}

Next Illuminate\Database\QueryException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' (SQL: update `tables` set `is_reserved` = 0 where (`0` = tables.restaurant_id and `1` = = and `2` = 1)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#4 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#5 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#6 [internal function]: Resources\Table\TableController->hold('1', '2')
#7 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#8 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#9 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#10 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[09-Jul-2021 23:07:24 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:492
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(492): PDO->prepare('update `tables`...')
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('update `tables`...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#6 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#7 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#8 [internal function]: Resources\Table\TableController->hold('1', '2')
#9 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#10 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#11 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#12 {main}

Next Illuminate\Database\QueryException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' (SQL: update `tables` set `is_reserved` = 1 where (`0` = tables.restaurant_id and `1` = = and `2` = 1)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#4 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#5 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#6 [internal function]: Resources\Table\TableController->hold('1', '2')
#7 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#8 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#9 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#10 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[09-Jul-2021 23:09:58 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:492
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(492): PDO->prepare('update `tables`...')
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('update `tables`...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#6 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#7 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#8 [internal function]: Resources\Table\TableController->hold('1', '2')
#9 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#10 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#11 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#12 {main}

Next Illuminate\Database\QueryException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' (SQL: update `tables` set `is_reserved` = 1 where (`0` = tables.restaurant_id and `1` = = and `2` = 1) and (`0` = tables.number and `1` = = and `2` = 2)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#4 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#5 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#6 [internal function]: Resources\Table\TableController->hold('1', '2')
#7 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#8 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#9 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#10 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[09-Jul-2021 23:10:54 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to a member function where() on int in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php:31
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#1 [internal function]: Resources\Table\TableController->hold('1', '2')
#2 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#3 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#4 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#5 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php on line 31
[09-Jul-2021 23:11:49 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to a member function where() on int in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php:31
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#1 [internal function]: Resources\Table\TableController->hold('1', '2')
#2 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#3 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#4 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#5 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Table\TableModel.php on line 31
[09-Jul-2021 23:13:47 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:492
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(492): PDO->prepare('update `tables`...')
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('update `tables`...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#6 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#7 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#8 [internal function]: Resources\Table\TableController->hold('1', '2')
#9 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#10 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#11 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#12 {main}

Next Illuminate\Database\QueryException: SQLSTATE[42S22]: Column not found: 1054 Unknown column '0' in 'where clause' (SQL: update `tables` set `is_reserved` = 1 where (`0` = tables.restaurant_id and `1` = = and `2` = 1) and (`0` = tables.number and `1` = = and `2` = 2)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('update `tables`...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(503): Illuminate\Database\Connection->run('update `tables`...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(436): Illuminate\Database\Connection->affectingStatement('update `tables`...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2913): Illuminate\Database\Connection->update('update `tables`...', Array)
#4 D:\XAMPP\htdocs\api\Resources\Table\TableModel.php(33): Illuminate\Database\Query\Builder->update(Array)
#5 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(23): Resources\Table\TableModel->hold(1, 2)
#6 [internal function]: Resources\Table\TableController->hold('1', '2')
#7 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#8 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#9 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#10 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[09-Jul-2021 23:16:44 Europe/Berlin] PHP Fatal error:  Uncaught BadMethodCallException: Call to undefined method Resources\Table\TableModel::release() in D:\XAMPP\htdocs\api\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('release')
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Eloquent\Model.php(1883): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'release', Array)
#2 D:\XAMPP\htdocs\api\Resources\Table\TableController.php(17): Illuminate\Database\Eloquent\Model->__call('release', Array)
#3 [internal function]: Resources\Table\TableController->release('1', '2')
#4 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#5 D:\XAMPP\htdocs\api\Core\Application.php(135): Core\Router->resolve()
#6 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#7 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\support\Traits\ForwardsCalls.php on line 50
[09-Jul-2021 23:49:15 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`nofipayn_restaurant_reservations_system`.`orders`, CONSTRAINT `orders_ibfk_3` FOREIGN KEY (`table_id`) REFERENCES `tables` (`id`)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(22): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`nofipayn_restaurant_reservations_system`.`orders`, CONSTRAINT `orders_ibfk_3` FOREIGN KEY (`table_id`) REFERENCES `tables` (`id`)) (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 2, [{"id":2}])) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(22): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[09-Jul-2021 23:58:02 Europe/Berlin] PHP Fatal error:  Uncaught Error: Object of class stdClass could not be converted to string in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:589
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(589): PDOStatement->bindValue(4, Object(stdClass), 2)
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(467): Illuminate\Database\Connection->bindValues(Object(PDOStatement), Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#7 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#9 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#10 [internal function]: Resources\Order\OrderController->create()
#11 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#12 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#13 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#14 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 589
[09-Jul-2021 23:59:09 Europe/Berlin] PHP Fatal error:  Uncaught Error: Object of class stdClass could not be converted to string in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:589
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(589): PDOStatement->bindValue(4, Object(stdClass), 2)
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(467): Illuminate\Database\Connection->bindValues(Object(PDOStatement), Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#7 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#9 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#10 [internal function]: Resources\Order\OrderController->create()
#11 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#12 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#13 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#14 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 589
[09-Jul-2021 23:59:47 Europe/Berlin] PHP Warning:  Object of class stdClass could not be converted to int in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 24
[09-Jul-2021 23:59:47 Europe/Berlin] PHP Warning:  Undefined array key "items" in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 27
[09-Jul-2021 23:59:47 Europe/Berlin] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php:27
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(27): count(NULL)
#1 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#2 [internal function]: Resources\Order\OrderController->create()
#3 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#4 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#5 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#6 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 27
[10-Jul-2021 00:06:01 Europe/Berlin] PHP Warning:  Object of class stdClass could not be converted to int in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 24
[10-Jul-2021 00:06:01 Europe/Berlin] PHP Warning:  Undefined array key "items" in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 27
[10-Jul-2021 00:06:01 Europe/Berlin] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php:27
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(27): count(NULL)
#1 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#2 [internal function]: Resources\Order\OrderController->create()
#3 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#4 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#5 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#6 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 27
[10-Jul-2021 00:12:27 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to undefined method stdClass::get() in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php:18
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#1 [internal function]: Resources\Order\OrderController->create()
#2 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#3 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#4 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#5 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 18
[10-Jul-2021 00:14:29 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to undefined method stdClass::get() in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php:18
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#1 [internal function]: Resources\Order\OrderController->create()
#2 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#3 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#4 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#5 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 18
[10-Jul-2021 00:15:39 Europe/Berlin] PHP Warning:  Object of class stdClass could not be converted to int in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 24
[10-Jul-2021 00:15:39 Europe/Berlin] PHP Warning:  Undefined array key "items" in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 27
[10-Jul-2021 00:15:39 Europe/Berlin] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php:27
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(27): count(NULL)
#1 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#2 [internal function]: Resources\Order\OrderController->create()
#3 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#4 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#5 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#6 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 27
[10-Jul-2021 00:17:27 Europe/Berlin] PHP Fatal error:  Uncaught Error: Cannot use object of type stdClass as array in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php:24
Stack trace:
#0 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#1 [internal function]: Resources\Order\OrderController->create()
#2 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#3 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#4 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#5 {main}
  thrown in D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php on line 24
[10-Jul-2021 00:17:50 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:17:51 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`nofipayn_restaurant_reservations_system`.`orders`, CONSTRAINT `orders_ibfk_3` FOREIGN KEY (`table_id`) REFERENCES `tables` (`id`)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`nofipayn_restaurant_reservations_system`.`orders`, CONSTRAINT `orders_ibfk_3` FOREIGN KEY (`table_id`) REFERENCES `tables` (`id`)) (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, 0)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:19:19 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:19:19 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:20:04 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:20:04 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(24): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:23:31 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:23:31 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:24:11 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:24:11 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:24:11 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:24:38 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:24:38 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:24:38 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:25:16 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:25:16 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:25:16 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:25:54 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:25:54 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:25:54 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:27:14 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:27:14 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:27:40 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:27:40 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:27:40 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:28:04 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:28:04 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:28:38 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:28:38 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[10-Jul-2021 00:29:03 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:29:03 Europe/Berlin] PHP Warning:  Undefined array key "id" in D:\XAMPP\htdocs\api\vendor\illuminate\collections\Collection.php on line 1414
[10-Jul-2021 00:29:03 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:471
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(471): PDOStatement->execute()
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('insert into `or...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#9 [internal function]: Resources\Order\OrderController->create()
#10 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#11 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#12 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#13 {main}

Next Illuminate\Database\QueryException: SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'table_id' cannot be null (SQL: insert into `orders` (`customer_id`, `restaurant_id`, `table_number`, `table_id`) values (100, 1, 3, ?)) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('insert into `or...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(472): Illuminate\Database\Connection->run('insert into `or...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(424): Illuminate\Database\Connection->statement('insert into `or...', Array)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Processors\Processor.php(32): Illuminate\Database\Connection->insert('insert into `or...', Array)
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2882): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `or...', Array, NULL)
#5 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(26): Illuminate\Database\Query\Builder->insertGetId(Array)
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(34): Resources\Order\OrderModel->insertOrder(Array)
#7 [internal function]: Resources\Order\OrderController->create()
#8 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#9 D:\XAMPP\htdocs\api\Core\Application.php(137): Core\Router->resolve()
#10 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#11 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
[11-Jul-2021 22:30:26 Europe/Berlin] PHP Warning:  Undefined array key "new_passowrd" in D:\XAMPP\htdocs\api\Resources\Auth\AuthModel.php on line 41
[11-Jul-2021 22:39:55 Europe/Berlin] PHP Warning:  Undefined array key "new_passowrd" in D:\XAMPP\htdocs\api\Resources\Auth\AuthModel.php on line 41
[11-Jul-2021 23:37:21 Europe/Berlin] PHP Warning:  Undefined variable $tables in D:\XAMPP\htdocs\api\Resources\Order\OrderController.php on line 153
[11-Jul-2021 23:38:50 Europe/Berlin] PHP Warning:  Undefined variable $tables in D:\XAMPP\htdocs\api\Resources\Order\OrderController.php on line 153
[11-Jul-2021 23:40:05 Europe/Berlin] PHP Warning:  Undefined variable $tables in D:\XAMPP\htdocs\api\Resources\Order\OrderController.php on line 153
[11-Jul-2021 23:40:36 Europe/Berlin] PHP Warning:  Undefined variable $tables in D:\XAMPP\htdocs\api\Resources\Order\OrderController.php on line 153
[11-Jul-2021 23:58:22 Europe/Berlin] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'order.is_finished' in 'where clause' in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:338
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(338): PDO->prepare('select `orders`...')
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(671): Illuminate\Database\Connection->Illuminate\Database\{closure}('select `orders`...', Array)
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('select `orders`...', Array, Object(Closure))
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(346): Illuminate\Database\Connection->run('select `orders`...', Array, Object(Closure))
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2313): Illuminate\Database\Connection->select('select `orders`...', Array, true)
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2301): Illuminate\Database\Query\Builder->runSelect()
#6 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2796): Illuminate\Database\Query\Builder->Illuminate\Database\Query\{closure}()
#7 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2302): Illuminate\Database\Query\Builder->onceWithColumns(Array, Object(Closure))
#8 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(67): Illuminate\Database\Query\Builder->get()
#9 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(159): Resources\Order\OrderModel->getNonFinished('orders.customer...', '46')
#10 [internal function]: Resources\Order\OrderController->getCustomerNonFinishedOrders('46')
#11 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#12 D:\XAMPP\htdocs\api\Core\Application.php(142): Core\Router->resolve()
#13 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#14 {main}

Next Illuminate\Database\QueryException: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'order.is_finished' in 'where clause' (SQL: select `orders`.*, `items`.`name` as `item_name`, `order_details`.`quantity` as `qty`, `order_details`.`total` as `total` from `orders` inner join `order_details` on `orders`.`id` = `order_details`.`order_id` inner join `items` on `order_details`.`meal_id` = `items`.`id` where `orders`.`customer_id` = 46 and `order`.`is_finished` = 1) in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php:678
Stack trace:
#0 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(638): Illuminate\Database\Connection->runQueryCallback('select `orders`...', Array, Object(Closure))
#1 D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php(346): Illuminate\Database\Connection->run('select `orders`...', Array, Object(Closure))
#2 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2313): Illuminate\Database\Connection->select('select `orders`...', Array, true)
#3 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2301): Illuminate\Database\Query\Builder->runSelect()
#4 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2796): Illuminate\Database\Query\Builder->Illuminate\Database\Query\{closure}()
#5 D:\XAMPP\htdocs\api\vendor\illuminate\database\Query\Builder.php(2302): Illuminate\Database\Query\Builder->onceWithColumns(Array, Object(Closure))
#6 D:\XAMPP\htdocs\api\Resources\Order\OrderModel.php(67): Illuminate\Database\Query\Builder->get()
#7 D:\XAMPP\htdocs\api\Resources\Order\OrderController.php(159): Resources\Order\OrderModel->getNonFinished('orders.customer...', '46')
#8 [internal function]: Resources\Order\OrderController->getCustomerNonFinishedOrders('46')
#9 D:\XAMPP\htdocs\api\Core\Router.php(170): call_user_func_array(Array, Array)
#10 D:\XAMPP\htdocs\api\Core\Application.php(142): Core\Router->resolve()
#11 D:\XAMPP\htdocs\api\index.php(33): Core\Application::run()
#12 {main}
  thrown in D:\XAMPP\htdocs\api\vendor\illuminate\database\Connection.php on line 678
